packages/opensrp-plans/src/ducks/jurisdictions/index.ts

```diff
@@ -0,0 +1,115 @@
+import { Dictionary } from '@onaio/utils/dist/types/types';
+import {
+  fetchActionCreatorFactory,
+  reducerFactory,
+  removeActionCreatorFactory,
+} from '@opensrp/reducer-factory';
+import { Geometry } from 'geojson';
+import { get, values } from 'lodash';
+import { Store } from 'redux';
+import { createSelector } from 'reselect';
+
+export interface LocationTag {
+  id: number;
+  name: string;
+}
+
+/** The shape of a jurisdiction received from the OpenSRP API */
+export interface Jurisdiction {
+  id: string;
+  geometry?: Geometry;
+  properties: {
+    code?: string;
+    geographicLevel: number;
+    name: string;
+    parentId?: string;
+    status: string;
+    version: string | number;
+    OpenMRS_Id?: string;
+    externalId?: string;
+    username?: string;
+  };
+  locationTags: LocationTag[];
+  serverVersion: number;
+  type: Readonly<'Feature'>;
+}
+
+/** the reducer name */
+export const jurisdictionReducerName = 'Jurisdiction';
+
+/** Jurisdiction Reducer */
+export const JurisdictionsReducer = reducerFactory<Jurisdiction>(jurisdictionReducerName);
+
+// actions
+/** actionCreator returns action to to add Item records to store */
+export const fetchJurisdictions = fetchActionCreatorFactory<Jurisdiction>(
+  jurisdictionReducerName,
+  'id'
+);
+export const removeJurisdictions = removeActionCreatorFactory(jurisdictionReducerName);
+
+// selectors
+/** prop filters to customize selector queries */
+export interface Filters {
+  jurisdictionId?: string /** jurisdiction id */;
+  geoLevel?: number /** return jurisdictions with this geo level */;
+}
+
+/**
```

Comment:
The docstrings from here onwards seem odd.  Do we want a description for all params and returns?  Lets make a decision and keep it consistent.  As well as look into the odd isolated `-` characters